packages/element/src/serialize.js

```diff
@@ -508,9 +527,9 @@ export function renderComponent(
 /**
  * Serializes an array of children to string.
  *
- * @param {Array}   children      Children to serialize.
- * @param {?Object} context       Context object.
- * @param {?Object} legacyContext Legacy context object.
+ * @param {import('react').ReactNodeArray}  children        Children to serialize.
+ * @param {Object} [context]       Context object.
+ * @param {Object} [legacyContext] Legacy context object.
```

Comment:
I think the "columns" need to be realigned here.

```suggestion
 * @param {import('react').ReactNodeArray} children        Children to serialize.
 * @param {Object}                         [context]       Context object.
 * @param {Object}                         [legacyContext] Legacy context object.
```